src/components/shared/modals/DownloadParticipantsModal.tsx

```diff
@@ -122,7 +125,9 @@ export default function DownloadParticipantsModal({
     >
       <div>
         <h4>
-          <Trans>Download CSV of {tokenSymbol || t`token`} holders</Trans>
+          <Trans>
+            Download CSV of {tokenSymbolText({ tokenSymbol })} holders ---
```

Comment:
**question** is the `---` intended?

Reply:
Nah accident - me tweaking to see immediate feedback to make sure I was editing the right thing 😅